apps/src/p5lab/spritelab/commands/criterionCommands.js

```diff
@@ -419,31 +399,8 @@ export const commands = {
   // Returns true if any sprite has one of: ['moving_with_arrow_keys', 'driving_with_arrow_keys', 'draggable'].
   interactiveBehaviorFound() {
     const spriteIds = this.getSpriteIdsInUse();
-    let result = false;
-    for (let i = 0; i < spriteIds.length; i++) {
-      const currentBehaviors = this.getBehaviorsForSpriteId(i);
-      if (
-        currentBehaviors.some(behavior =>
-          [
-            'moving_with_arrow_keys',
-            'driving_with_arrow_keys',
-            'draggable',
-          ].includes(behavior)
-        )
-      ) {
-        result = true;
-      }
-    }
-    return result;
-  },
-
-  // Special function for lesson: Mini-Project: Collector Game
-  // Returns true if any sprite has one of the "non-interactive" behaviors.
-  itemBehaviorFound() {
```

Comment:
why could this function be removed?

Reply:
Oh! Meant to leave a comment explaining this. It's removed for two reasons:
* it's an exact copy of `interactiveBehaviorFound`
* it's not used in any levels